config/operator/operator/operator.yaml

```diff
@@ -43,7 +43,7 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 30Mi
+            memory: 100Mi
```

Comment:
any reason not to go best effort and use only requests? the memory depends on how many objects live in the cluster which varies by size.

Reply:
Good point